cookbooks/bach_repository/attributes/default.rb

```diff
@@ -6,7 +6,11 @@
 # was made by the old build_bins.sh.  For now, we are just replicating
 # the behavior of the old script.
 #
-default[:bach][:repository][:repo_directory] = '/home/vagrant/chef-bcpc'
+
+
+user = ENV['SUDO_USER'] || ENV['USER'] || 'vagrant'
```

Comment:
This gets copy-pasted a couple of times.  It should probably be defined once in a single place.